meta-xt-control-domain/recipes-extended/qemu-restart-monitor/files/qemu-restart-monitor-ExecStart.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "QEMU restart monitor started!";
+
+# Fetch DomD Xen domain identifier
+DOMD_ID="";
+
+while [ "$DOMD_ID" == "" ]; do
+    DOMD_ID=$(xl list | awk '{ if ($1 == "DomD") print $2 }');
```

Comment:
Hi @lorc, I'll close this point as of now. In case of usage of the "xl domid" command, I'll need to additionally check for the error state (which happens when the domain is not available ) to void assigning the error content to the DOMD_ID variable.

The error looks like this:

```
root@generic-armv8-xt-dom0:~# xl domid non_existing_domain_id
Can't get domid of domain name 'non_existing_domain_id', maybe this domain does not exist.
```

It'll take more lines of code and won't be that elegant.

awk, in its turn, will return empty string in case of non-existing domain.